static/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Themeroller</title>
```

Comment:
Why did you pull this in already? That's a totally valid point and there's no hurry.